packages/backend/src/models/SavedChartModel.ts

```diff
@@ -999,10 +999,21 @@ export class SavedChartModel {
                         })
                         .from(SavedChartsTableName)
                         .leftJoin(
-                            SpaceTableName,
-                            'saved_queries.space_id',
-                            'spaces.space_id',
+                            DashboardsTableName,
+                            `${DashboardsTableName}.dashboard_uuid`,
+                            `${SavedChartsTableName}.dashboard_uuid`,
                         )
+                        .innerJoin(SpaceTableName, function spaceJoin() {
+                            this.on(
+                                `${SpaceTableName}.space_id`,
+                                '=',
+                                `${DashboardsTableName}.space_id`,
+                            ).orOn(
+                                `${SpaceTableName}.space_id`,
+                                '=',
+                                `${SavedChartsTableName}.space_id`,
+                            );
+                        })
```

Comment:
yeah okay, after thinking for a minute or two or three, I get it. nice

Reply:
Noice! There are some other instances of this join in the same file, it was just missing from this query 💪